Add KernelCare Clients for Debian/Ubuntu (#4000)
theforeman/foreman-documentation: guides/common/modules/ref_kernelcare-client-repositories.adoc

:_mod-docs-content-type: REFERENCE

[id="kernelcare-client-repositories"]
= KernelCare client repositories

> The following tables list KernelCare client repositories of various supported client operating systems.

Reviewer: ```suggestion
The following tables list KernelCare client repositories for supported client operating systems.
```

Author: Applied.

Each table corresponds to a product name and each row corresponds to a repository name.

.Prerequisites
* Import the GPG public into {Project}.
** For Yum content, import: https://repo.cloudlinux.com/kernelcare/RPM-GPG-KEY-KernelCare
** For Deb content, import https://repo.cloudlinux.com/kernelcare/kernelcare.gpg
+
For more information, see {ContentManagementDocURL}Importing_a_Custom_GPG_Key_content-management[Importing a GPG key] in _{ContentManagementDocTitle}_.

> [id="kernelcare-client-almalinux-9"]

Reviewer 2: confirm
